web/codegen.yml

```
@@ -0,0 +1,36 @@
+overwrite: true
+schema: http://localhost:8080/api/graphql
```

```suggestion
schema: ../server/schema.graphql
```

The backend is in the same repository, so you can retrieve the schema locally!